app/layout.jsx

```diff
@@ -37,6 +38,7 @@ export default function RootLayout({ children }) {
         <Navbar />
         <main id="top">{children}</main>
         <Footer />
+        <FooterBlogList />
```

Comment:
Blog list needs to be in the Footer, within the footer tag. Crawlers and users may ignore if ti comes after footer.